### ISSUE TITLE
Example of using websocket-kit with system proxy?

### ISSUE THREAD

**Issue by bunchjesse** [vapor/websocket-kit#110]
Do you have an example anywhere of how to connect to a web socket server via the system's proxy (macOS Big Sur)?

- I'm trying to debug some of the communication using Proxyman.
  - The web socket requests do not seem to be going through the proxy.
  - Normal requests made via Foundation show up just fine.
- I've scoured the docs and existing issues in this repository but haven't found what I'm looking for.

```
try await WebSocket.connect(to: url, on: eventLoopGroup) { socket in
  ...
}
```

**Comment by 0xTim** [comment]
Vapor's WebsocketKit does not currently support connecting to a websocket server via a proxy. It's something we'd need to add and could add by adding a proxy channel handler to our bootstrap (as hinted at [here](https://forums.swift.org/t/how-to-get-websocket-client-to-use-proxy/41038))

**Comment by 0xTim** [comment]
This should be solved by #130
